src/test/java/com/github/caldav4j/CalDAVCollectionTest.java

```diff
@@ -212,39 +198,37 @@ public void testGetCalendarByPath() throws Exception {
 	 */
 	@Test
 	public void testGetEventResources() throws Exception {
-		Date beginDate = ICalendarUtils.createDateTime(2006, 0, 1, null, true);
-		Date endDate = ICalendarUtils.createDateTime(2006, 0, 9, null, true);
-		Thread.sleep(1000);
-		List<Calendar> l = collection.getEventResources(fixture.getHttpClient(),
-				beginDate, endDate);
+		Date beginDate = ICalendarUtils.createDateTime(2006, java.util.Calendar.JANUARY, 1, null, true);
+		Date endDate = ICalendarUtils.createDateTime(2006, java.util.Calendar.JANUARY, 9, null, true);
+		
+		GenerateQuery gq = new GenerateQuery();
+		gq.setFilter("VEVENT");
+		gq.setTimeRange(beginDate, endDate);		
+		
+		List<Calendar> l = collection.queryCalendars(fixture.getHttpClient(),  gq.generate());
 
 		for (Calendar calendar : l) {
-			ComponentList vevents = calendar.getComponents().getComponents(
+			ComponentList<VEvent> vevents = calendar.getComponents().getComponents(
 					Component.VEVENT);
-			VEvent ve = (VEvent) vevents.get(0);
+			VEvent ve = vevents.get(0);
 			String uid = ICalendarUtils.getUIDValue(ve);
 			int correctNumberOfEvents = -1;
-			if (ICS_DAILY_NY_5PM_UID.equals(uid)) {
+			if (ICS_GOOGLE_DAILY_NY_5PM_UID.equals(uid)) {
 				// one for each day
 				correctNumberOfEvents = 1;
-			} else if (ICS_ALL_DAY_JAN1_UID.equals(uid)) {
+			} else if (ICS_GOOGLE_ALL_DAY_JAN1_UID.equals(uid)) {
 				correctNumberOfEvents = 1;
-			} else if (ICS_NORMAL_PACIFIC_1PM_UID.equals(uid)) {
+			} else if (ICS_GOOGLE_NORMAL_PACIFIC_1PM_UID.equals(uid)) {
 				correctNumberOfEvents = 1;
-			} else if (ICS_FLOATING_JAN2_7PM_UID.equals(uid)) {
+			} else if (ICS_GOOGLE_FLOATING_JAN2_7PM_UID.equals(uid)) {
```

Comment:
Why did you have to change the test?

Reply:
Because I unified the ICS (the contents were almost identical), due to that purge I removed the constants used in this test and replaced them with the new ones.